tests/foreman/ui/test_rhcloud_inventory.py

```diff
@@ -108,7 +123,7 @@ def test_rhcloud_inventory_e2e(
         report_path = session.cloudinventory.download_report(org.name)
         inventory_data = session.cloudinventory.read(org.name)
     # Verify that generated archive is valid.
-    common_assertion(report_path, inventory_data, org, module_target_sat)
+    common_assertion(report_path, inventory_data, org, module_target_sat, subscription_setting)
```

Comment:
We have other tests that call common_assertion in this file, do we need to pass in true for those as well?

Reply:
No as the subscription_setting is set to True by default in the common_assertion